snippets/card-product.liquid

```diff
@@ -479,7 +479,7 @@
                           {% endif %}
                           class="link product__view-details animate-arrow small-hide medium-hide"
                         >
-                          {{ 'products.product.view_full_details' | t }}
+                          <span class="link__text">{{ 'products.product.view_full_details' | t }}</span>
```

Comment:
Here I don't think we need the extra span element and class. We can keep the text as it was originally and use some whitespace control within the brackets: 
```
{{- 'products.product.view_full_details' | t -}}
```